### ISSUE TITLE
Config file not correctly read

### ISSUE THREAD

**Issue by pixelastic** [algolia/create-instantsearch-app#36]
Hello,

I pulled master and ran `yarn create instantsearch-app my-app --config config.json`, with `config.json` being:

```json
{
  "name": "TalkSearch",
  "template": "InstantSearch.js",
  "libraryVersion": "2.8.0",
  "appId": "8J7GPSC0XN",
  "apiKey": "28a29bbca9c6d43514d6654de6a8471c",
  "indexName": "dotconferences",
  "searchPlaceholder": "Search for topic, speaker or any sentence",
  "mainAttribute": "video.title",
  "attributesForFaceting": ["conference.year", "speaker.name"]
}
```

But, the prompt is still asking me all the questions (appID, indexName, etc) and is taking those into account. It seems like the config file is not read.

**Comment by francoischalifour** [comment]
Hi, you'll need to use the `beta` flag for now. Sorry, I didn't document that.

```
yarn create instantsearch-app@beta my-app --config config.json
```

Let me know if that works!

**Comment by francoischalifour** [comment]
Closing since version 3.0.0 has been released (d8bd9987503a4670c1919e8a2940e9e094b87f8e)
